x-pack/plugins/reporting/server/routes/generate_from_savedobject_immediate.ts

```diff
@@ -64,12 +63,8 @@ export function registerGenerateCsvFromSavedObjectImmediate(
       const runTaskFn = runTaskFnFactory(reporting, logger);
 
       try {
-        const jobDocPayload: ScheduledTaskParamsPanelCsv = await scheduleTaskFn(
-          jobParams,
-          req.headers,
-          context,
-          req
-        );
+        // FIXME: no scheduleTaskFn for immediate download
+        const jobDocPayload = await scheduleTaskFn(jobParams, req.headers, context, req);
```

Comment:
This line is what creates the complexity with getting Reporting to work with Task Manager. This CSV export handler uses the same methods other export types use for queuing, but that bind needs to be separated since this handler is responsible for "immediate" download of CSV without queuing.